_posts/2025-04-21-OpenSearch-Benchmark---The-Journey-Ahead.md

```diff
@@ -0,0 +1,62 @@
+
+
+## The OSB Project
+
+At the same time that OpenSearch was forked off from Elasticsearch, OSB was published as a fork of Rally. The project was quiescent for a short while, but starting a couple of years ago, it took off with a large number of improvements being added and bugs getting fixed, and a regular release cadence. The tool has since become significantly more robust, reliable and easy to use. New workloads have been added such as [`big5`](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/big5) for comparing search performance, and others in domains such as [vector search](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/vectorsearch), [neural search](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/neural_search) and hybrid search, which are relevant for generative AI and machine learning applications.
+
+The OSB team is also responsible for publishing [authoritative benchmark results](https://benchmarks.opensearch.org) that are relied upon by the community to track OpenSearch performance.
+
+
+### Why Use OSB?
+
+Not to be facetious, but, in a nutshell, because you have few other options. If you are operating an OpenSearch deployment for instance, the easiest and quickest way to carry out an end-to-end system test would be to run an OSB workload.
+
+Moving ahead from that basic level of testing onward, to simulate scenarios in domains ranging from log analytics to search, to run queries in categories such as sorts, aggregations and date histograms and to exercise neural search and k-NN algorithms, OSB is an application that evokes the functionality of the Swiss army knife. It can be used to size clusters, tune for the appropriate shard count, gain assurance that upgrading to a new version will not degrade performance and even encapsulate production workloads.
+
+Independent third parties such as [_Trail of Bits_](https://www.trailofbits.com/) have used OSB as their preferred benchmarking tool to carry out [comparative performance tests of OpenSearch against Elasticsearch](https://blog.trailofbits.com/2025/03/06/benchmarking-opensearch-and-elasticsearch/).
+
+
+### How Does One Use OSB?
+
+The easiest way to get started with OSB is to begin with the [Quick Start](https://docs.opensearch.org/docs/latest/benchmark/quickstart/) guide. Although the reference sections are currently yet being enhanced, the [User's Guide](https://docs.opensearch.org/docs/latest/benchmark/user-guide/index/) is eminently usable and should get a user proficient in using OSB in short order.
```

Comment:
Second sentence

The [User's Guide]<link> is currently under development, but still a useful reference to get started.

Reply:
Updated.